modules/moa-installing-prerequisites.adoc

```diff
@@ -0,0 +1,217 @@
+  delete      Delete a specific resource
+  describe    Show details of a specific resource
+  edit        Edit a specific resource
+  help        Help about any command
+  init        Applies templates to support Managed OpenShift on AWS clusters
+  list        List all resources of a specific type
+  login       Log in to your Red Hat account
+  logout      Log out
+  logs        Show logs of a specific resource
+  verify      Verify resources are configured correctly for cluster install
+  version     Prints the version of the tool
+
+Flags:
+      --debug     Enable debug mode.
+  -h, --help      help for moactl
+  -v, --v Level   log level for V logs
+
+Use "moactl [command] --help" for more information about a command.
+----
++
+.. (Optional) You can run `moactl completion` to generate a bash completion file. Add this file to the correct location for your operating system. For example, on a Linux machine run the following command to enable moactl bash completion:
++
+[source, bash]
+----
+$ moactl completion > /etc/bash_completion.d/moactl
+
+source /etc/bash_completion.d/moactl
+----
++
+. Run the following command to verify that your AWS account has the necessary permissions:
++
+[source,bash]
+----
+$ moactl verify permissions
+
+I: Validating SCP policies...
+I: AWS SCP policies ok
+----
++
+.. Verify that your AWS account has the necessary quota to deploy an OpenShift cluster.
++
+[source,bash]
+----
+$ export AWS_DEFAULT_REGION=us-west-2 && moactl verify quota
+
+I: Validating AWS quota...
+E: Insufficient AWS quotas
+E: Service ebs quota code L-FD252861 Provisioned IOPS SSD (io1) volume storage not valid
+----
++
+[NOTE]
+====
+Sometimes quota varies by region, which may prevent you from deploying. If you receive any errors, try a different region.
+====
++
+If you need to increase your quota, navigate to your link:https://aws.amazon.com/console/[AWS console], and request a quota increase for the service that failed.
++
+Once both the permissions and quota checks pass, proceed to initializing your AWS account.
++
+. Prepare your AWS account for cluster deployment
++
+.. Run the following command to log in to your Red Hat account with moactl. Replace `<my-offline-access-token>` with your token:
++
+[NOTE]
+====
+If you do not already have a Red Hat account, link:https://cloud.redhat.com/[create one here]. Be sure to accept the required terms and conditions. Then, check your email for a verification link.  
+
+After creating your Red Hat account, follow this link to link:https://cloud.redhat.com/openshift/token/moa[get an offline access token].
+====
++
+[source, bash]
+----
+$ moactl login --token="<my-offline-access-token>"
+----
++
+.. Run the following command to verify your Red Hat and AWS credentials are setup correctly.  Check that your AWS Account ID, Default Region and ARN match what you expect.  You can safely ignore the rows beginning with OCM for now (OCM stands for OpenShift Cluster Manager).
```

Comment:
I've never seen this construction:

> (OCM stands for OpenShift Cluster Manager).

I'd expand for the first usage in every module, and then use the abbreviation as needed.

Reply:
I was concerned here b/c OCM is still not really customer facing, but it's all over the output of this command.